ci-operator/step-registry/aws/provision/route53/private-hosted-zone/aws-provision-route53-private-hosted-zone-commands.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+curl -L https://github.com/mikefarah/yq/releases/download/3.3.0/yq_linux_amd64 -o /tmp/yq && chmod +x /tmp/yq
+
+export AWS_SHARED_CREDENTIALS_FILE="${CLUSTER_PROFILE_DIR}/.awscred"
+REGION="${LEASED_RESOURCE}"
+CONFIG="${SHARED_DIR}/install-config.yaml"
+CLUSTER_NAME="$(/tmp/yq r "${CONFIG}" 'metadata.name')"
+
+route53_hosted_zone_name="${CLUSTER_NAME}.${BASE_DOMAIN}"
```

Comment:
actually, the cluster name is `${NAMESPACE}-${JOB_NAME_HASH}`, using this will make this step independent, whatever `install-config.yaml` is generated or not while running this step, you can refer to [1] [2]


[1] https://github.com/openshift/release/blob/master/ci-operator/step-registry/aws/provision/bastionhost/aws-provision-bastionhost-commands.sh#L22
[2] https://github.com/openshift/release/blob/master/ci-operator/step-registry/aws/provision/vpc/shared/aws-provision-vpc-shared-commands.sh#L24

Reply:
Done